components/core/Dockerfile

```diff
@@ -1,32 +1,62 @@
-FROM python:3-alpine as builder
+# Using slim version as base
+FROM python:3.7-slim as base
 
+# Setting current work directory to /home/Bassa
 WORKDIR /home/Bassa
+
+# Copying API component
 COPY . .
 
-RUN ls && pip install pylint \
-&& pylint -E .
+# Installing dependencies
+RUN apt-get update && \ 
+    apt-get install -y --no-install-recommends python3-pip build-essential && \
+    apt-get install -y default-libmysqlclient-dev python3-dev && \
```

Comment:
Do we need another `apt-get install`? Can't we put these stuff also into the first `apt-get install`?

Reply:
Sure we can do that 👍.  Will install them all under one single command

> Do we need another `apt-get install`? Can't we put these stuff also into the first `apt-get install`?

Reply:
Done 😃!